src/main/java/com/uber/cadence/internal/worker/WorkflowWorker.java

```diff
@@ -250,23 +240,35 @@ private void sendReply(
 
   private class DecisionTaskWithHistoryIteratorImpl implements DecisionTaskWithHistoryIterator {
 
-    private long start = System.currentTimeMillis();
+    private final Duration RetryServiceOperationInitialInterval = Duration.ofMillis(200);
```

Comment:
Field and method names in Java are always start from lower case letter. The exception is CONSTANTS_THAT_USE_THIS_STYLE

Reply:
Resovled